apps/api/src/shared/auth/auth.guard.ts

```diff
@@ -49,10 +49,10 @@ export class AuthGuard implements CanActivate {
   }
 
   private async assignIdentityIdToRequest(request: Request): Promise<void> {
-    const email = request.auth?.payload['custom_email_claim'] as string;
-    if (!email) throw new UnauthorizedException();
+    const sub = request.auth?.payload.sub as string;
```

Comment:
Do we need `as string` for sub property?

Reply:
No, we do not. I will remove it.